internal/pkg/cli/cmd/template/test/create.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/helpmsg"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	createOp "github.com/keboola/keboola-as-code/pkg/lib/operation/template/local/test/create"
+)
+
+func CreateCommand(p dependencies.Provider) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [template] [version]",
+		Short: helpmsg.Read(`template/test/create/short`),
+		Long:  helpmsg.Read(`template/test/create/long`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			d, err := p.DependenciesForLocalCommand()
+			if err != nil {
+				return err
+			}
+
+			if len(args) < 1 {
+				return fmt.Errorf(`please enter argument with the template ID you want to use and optionally its version`)
+			}
+
+			// Get template repository
+			repo, _, err := d.LocalTemplateRepository(d.CommandCtx())
+			if err != nil {
+				return err
+			}
+
+			// Optional version argument
+			var versionArg string
+			if len(args) > 1 {
+				versionArg = args[1]
+			}
+
+			tmpl, err := d.Template(d.CommandCtx(), model.NewTemplateRef(repo.Ref(), args[0], versionArg))
```

Comment:
Optional: mohol by si niekde vyssie v kode pridat `templateId := args[0]`, aby bolo hned jasne co je to za hodnota.